pysonnet/projects.py

```diff
@@ -405,7 +407,103 @@ def make_sonnet_file(self, file_path, clean=True):
         if not os.path.isdir(subfolder):
             os.mkdir(subfolder)
         log.debug("geometry project saved")
-
+   
+	def current_desnity_xml(**kwargs):
+		
+    	# xml file name
+    	name = kwargs.get('name', 'test')
+
+    	# define filename to be exported
+    	filename = kwargs.get('filename', "current1")
+
+    	# define the sonnet file whose data we want to access
+    	label = kwargs.get('label', "current1.son")
+
+    	# entire sonnet bounding box = "Whole"
+    	Region_style = kwargs.get('Region_style', "Whole")
+
+    	# current density level
+    	Levels_stop = kwargs.get('Levels_stop', "0")
+    	Levels_range = kwargs.get('Levels_range', "Some")
+    	Levels_start = kwargs.get('Levels_start', "0")
+
+    	# grid size for each cell
+    	Grid_x_step = kwargs.get('Grid_x_step', "10")
```

Comment:
To stay consistent with python's style guide (pep8), all variable names should be lower case